app/src/androidTest/java/com/github/orkest/SignInTest.kt

```diff
@@ -0,0 +1,34 @@
+package com.github.orkest
```

Comment:
Nice tests, an idea would be to add a test to check if the logo image is indeed displayed as expected on the UI

Reply:
thanks I'll add this